### PR TITLE
Add a section to the documentation about how to run the docker-based …

### PR DESCRIPTION
**Pull request by cpennington**
…devstack service

@zubair-arbi @clintonb: Review?

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -105,3 +105,25 @@ OIDC client via LMS admin.
     $ python manage.py runserver 8150
 
 .. _Django's runserver command: https://docs.djangoproject.com/en/1.8/ref/django-admin/#runserver-port-or-address-port
+
+
+Run a devstack cluster
+----------------------
+If you don't want to navigate all of the dependent services manually,
+you can use the `docker-based`_ devstack. There are two make targets
```
Comment (clintonb) [inline docs/getting_started.rst]: Capitalize: Docker

**Comment by cpennington** [comment]
@zubair-arbi: I'd love to hear any comments you have about where these docs and the docs I just added to config (https://github.com/edx/configuration/pull/2696) could use expansion to help resolve whatever issues you ran into.

**Comment by clintonb** [comment]
:+1:

**Comment by zubair-arbi** [comment]
lgtm :+1: 

@cpennington I would also like to add some more documentation either for this project or in general, something like:

To start the credentials docker container you will need the docker image with the same tag for credentials, available at docker hub 'https://hub.docker.com/r/edxops/' as mentioned in the file 'docker-compose.yml' in this project, e.g. 'image: credentials:v1'.

You can also create your own image locally by following these steps:
1. Checkout the edx configrutation repo which has the credentials role.
2. Pull the latest trusty-common image from docker hub as a base image to create the credentials image with command'docker pull edxops/trusty-common:latest'.
3. Start creating credentials image with the command 'docker build -t credentials:latest -f docker/build/credentials/Dockerfile .'
Note: This will fetch the credentials code from git repo.
4. Confirm the image has been created by listing the local docker images with command 'docker images'.

Once the credentials image is created now all you need is to checkout or move to an existing local credentials directory and use the make commands 'make start-devstack' to start the docker container.
You can see the up container by using command 'docker ps'.

**Comment by cpennington** [comment]
@zubair-arbi I added a section on local development. Does that cover it?

**Comment by feanil** [comment]
Closing since I think this is superseded by docker devstack.

**Pull request closed without merging**